pwa-builder/pwa-starter
# Enable widget

Enable widget by adding:
- widget member in manifest
- widget events in service worker
- adaptive card widget template and data
- widget screenshot

Testing steps:
1. Install dependencies and run `npm start`
2. Install PWA
3. Win+W to open widget board
4. Add "Starter Widget" widget to board

System requirement:
- In Microsoft Edge, go to edge://flags, enable "Wb App Widgets" and "Experimental Web Platform features"
- Ensure Windows Developer mode is turned on
- Ensure WinAppSDK 1.2+ is installed
